cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file is a placeholder template when initializing Cypress and doesn't appear to be used. Consider removing it?